atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1308,48 +1287,42 @@ protected ConflictHandler getConflictHandlerForTable(TableReference tableRef) {
             tableRef);
     }
 
-    private String getExpiredLocksErrorString(@Nullable LockRefreshToken commitLocksToken,
-                                              Set<LockRefreshToken> expiredLocks) {
+    private String getExpiredLocksErrorString(@Nullable LockTokenV2 commitLocksToken,
+                                              Set<LockTokenV2> expiredLocks) {
         return "The following external locks were required: " + externalLocksTokens
             + "; the following commit locks were required: " + commitLocksToken
             + "; the following locks are no longer valid: " + expiredLocks;
     }
 
-    private void throwIfExternalAndCommitLocksNotValid(@Nullable LockRefreshToken commitLocksToken) {
-        Set<LockRefreshToken> expiredLocks = refreshExternalAndCommitLocks(commitLocksToken);
+    private void throwIfPreCommitRequirementsNotMet(@Nullable LockTokenV2 commitLocksToken) {
+        Set<LockTokenV2> expiredLocks = refreshExternalAndCommitLocks(commitLocksToken);
```

Comment:
Thanks for the explanation! I think renaming would be helpful for readability here 👍